### DISCUSSION TITLE
Where is the default `data-dir`?

### DISCUSSION THREAD

**Discussion by eccentricexit** [erigontech/erigon#3062] [Q&A] [answered]
Just finished the full sync of my erigon node and I want to start the rpc daemon.

Unfortunately I don't know how to fill the `--data-dir` flag it requires to use the local db. 

What is the default path to data-dir?

**Comment by eccentricexit** [comment, marked as answer]
Answering my own question, its in `~/.local/share/erigon`

**Comment by AskAlexSharov** [comment]
https://wiki.archlinux.org/title/XDG_Base_Directory
